add Heatsink Fischer FK24413DPAK

hvraven opened the pull request:
a slightly compressed version of the already existing Heatsink_Fischer_FK24413D2PAK_26x13mm. [Datasheet](https://www.fischerelektronik.de/pim/upload/fischerData/cadpdf/base/fk_244_13_d_pak.pdf). Differences/similarity can be seen in [this overview](https://docs-emea.rs-online.com/webdocs/0d94/0900766b80d946b1.pdf).

![Screenshot_20190802_183229](https://user-images.githubusercontent.com/1010629/62384932-ed520800-b553-11e9-8291-94f053153c95.png)

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the footprint(s) you are contributing
- [x] An example screenshot image is very helpful 
- [x] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

myfreescalewebpage commented:
Hi @hvraven , thanks for contributing,

A few comments I have during my review:

- [ ] Not sure why the silkscreen is outside of the CrtYd layer on the two footprint you mentioned, is there a reason for that ? As far as I know for this kind of footprint 0.12mm between F.Fab and F.SilkS and 0.5mm between F.Fab and F.CrtYd is enough
- [ ] 3D model name to be fixed (see travis error)

Cheers,
Joel

hvraven commented:
changed the silk and courtyard to the standard values and fixed the 3d model name. Now looks like this:
![Screenshot_20200922_131811](https://user-images.githubusercontent.com/1010629/93875916-32944480-fcd6-11ea-8c35-64a1daf731b4.png)

myfreescalewebpage commented:
Thanks, no more comment, merging.
